AnalysisReporting/analysisreport/templates/analysisreport/loginpage.html

```diff
@@ -0,0 +1,113 @@
+ 
+
+{% load static %}
+{% csrf_token %}
+<!DOCTYPE html>
+<html>
+<head>
+	<title>login</title>
+
+	<meta charset="utf-8">
+    <title> Signup </title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <script src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/jquery.validation/1.15.0/jquery.validate.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/jquery.validation/1.15.0/additional-methods.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="{% static 'analysisreport/bootstrap-3.3.7-dist/css/bootstrap.min.css' %}" />
+    <link rel="stylesheet" type="text/css" href="{% static 'analysisreport/bootstrap-3.3.7-dist/css/bootstrap-theme.min.css' %}" />
+    
+     <script src="{% static 'analysisreport/bootstrap-3.3.7-dist/js/bootstrap.min.js' %}"></script>
+     <script src="{% static 'analysisreport/js/jquery.validate.js' %}"></script>
+
+ <script src="{% static 'analysisreport/js/validations.js' %}"></script>
+
+
+<script src="analysisreport/js/Cookie/jquery.cookie.js"></script>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+   
+
+
+
+</head>
+<body>
+
+
+ <form method="POST">{% csrf_token %}
+
+ <header style="  background-color: grey; padding: 10px;">
+  <h1 style="height: 20px; margin-top: -2px;  ">Analysis & Reporting</h1>      
+  </header>
+
+
+
+ <div class="container" style="text-align:center;background-color: lightgrey; "> 
+
+ <h3 style="text-align:center;"> Login </h3>
+ 
+<br>
```

Comment:
do not use <br> tag spacing and margin purpos. use padding and margin properties from CSS.